components/Components/StellrLink.tsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import Link from 'next/link';
+import React from 'react';
+
+const StellrLink = (props:any) => {
```

Comment:
Create a type file for props

You can follow the component structure.

Ariallabel: string
Children: React.Reactnode